src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+
+public class Duke {
+    private TaskList tasks;
+//    private String filePath = "data/tasks.txt";
+//    private String directoryName = "data";
+
+    private Storage storage;
+    private Ui ui;
+
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() {
+        try {
+            storage.load();
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+
+        ui.showWelcome();
+        Scanner scanner = new Scanner(System.in);
+        while(true) {
+            ui.showLine();
+            String command = scanner.nextLine();
+            try {
+                if (command.equals("bye")) {
+                    ui.showBye();
+                    break;
+                }
+                if (command.equals("list")) {
+                    tasks.printList();
+                    continue;
+                }
+                String[] parsedCommand = Parser.parseCommand(command);
+
+                switch (parsedCommand[0]) {
+                case "done":
+                    //Marks tasks as done
```

Comment:
```suggestion
// Marks tasks as done
```
While this may not be specifically stated in the coding standards, I believe comments look nicer with a space after the `//`.
Likewise for the other inline comments.